manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -95,6 +95,8 @@ spec:
         {{- else }}
           value: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}.{{ .Values.global.istioNamespace }}.svc:15012
         {{- end }}
+        - name: OUTPUT_CERTS # Store certs in non-root directory.
```

Comment:
We should not need this

Reply:
Without this we get permission denied when the certs are generated

Reply:
Then something else is wrong. A waypoint is like 99% similar to a gateway, we should not need this.